package.json

```diff
@@ -18,32 +18,38 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "mocha",
-    "prepublish": "browserify -o browser.js -e index.js"
```

Comment:
I'm unconcerned with publishing a pre-baked browser build and am totally fine with removing it. When I made this comment I was just noticing the change here and didn't notice that the file had been fully dropped from the repo.